en-US/Design.xml

```diff
@@ -15,15 +15,20 @@
 		 <formalpara id="capitalization">
 				<title>Capitalization</title>
 					<para>
-						The standard for all Red&nbsp;Hat technical documentation is title case for all headings and titles.
+					The standard for all Red&nbsp;Hat technical documentation is title case for all headings and titles.
             Table headings, procedure headings, and formal paragraph titles fall under this heading, and consequently, standard title case capitalization rules apply.
-            The currently accepted reference for determining title case is at <ulink url="https://titlecase.com/titlecase">https://titlecase.com/titlecase.</ulink>
 					</para>
-		  </formalpara>
-					 <para>
-						Use sentence case for captions, legends, diagram labels, and table column headers.
+		</formalpara>
+			<para>
+			Use title case also when referring to the titles of other publications, even if the title on the publication itself uses different casing.
+			</para>					
+			<para>
+			The currently accepted reference for determining title case is at <ulink url="https://titlecase.com/titlecase">https://titlecase.com/titlecase.</ulink>
```

Comment:
I forget what the convention is for including periods or not in URLs. Good idea to check.

Reply:
Corrected URL to move period outside it.